test/data/extension-compat-test-extension/blocking.html

```diff
@@ -0,0 +1,99 @@
+<html>
```

Comment:
Copy of blocking.html in test/data with some additions for cookies.  I intentionally kept some functions that are unused like xhr because we might use them for more tests later.